src/test/pmem2_deep_flush/pmem2_deep_flush.c

```diff
@@ -108,6 +108,18 @@ FUNC_MOCK_RUN_DEFAULT {
 	return 1;
 }
 FUNC_MOCK_END
+
+/*
+ * read -- read mock
+ */
+FUNC_MOCK(read, int, int fd, void *buffer, size_t nbytes)
+FUNC_MOCK_RUN_DEFAULT {
+	UT_ASSERTeq(nbytes, 2);
+	UT_ASSERTeq(fd, MOCK_FD);
+
```

Comment:
`memset(buffer, ???, nbytes);` ?
Otherwise the callers will read uninitialized data.